toolchain.py

```diff
@@ -136,13 +139,16 @@ def remove_all(self, prefix):
         self.sync()
 
     def sync(self):
-        # http://stackoverflow.com/questions/12309269/write-json-data-to-file-in-python/14870531#14870531
+        # http://stackoverflow.com/questions/12309269/write-json-data-to-file-
+        # in-python/14870531#14870531
```

Comment:
Minor suggestion in order not to cut a web link:
```suggestion
        # https://stackoverflow.com/a/14870531/185510
```